pkg/bundle/bundle_test.go

```diff
@@ -1215,14 +1160,17 @@ func Test_Reconcile(t *testing.T) {
 			},
 			expEvent: `Normal Synced Successfully synced Bundle to all namespaces`,
 		},
-		"if Bundle had a Secret target, and Secret targets are disabled, return an error": {
+		"if Bundle has Secret target, and Secret targets are disabled, return an error": {
```

Comment:
@erikgb This test was testing that all secrets are removed if the resource had a Secret target.
Now, Bundles that pointed to Secrets will not cleanup ConfigMaps if the secrets are being disabled.
Not sure if this is a problem, but I just want to make sure you note this.

Reply:
So a user will disable secret targets and modify the Bundle _at the same time_? If you disable secret targets, but leave the  bundle unchanged, it will error out. If you update the bundle and then disable secret targets, the secrets should already be deleted? Or are you thinking of a scenario I don't see? In that case, we could reintroduce some state under status, but I would prefer to avoid it.